astro/src/content/docs/apis/_user-response-body-base.mdx

```diff
@@ -0,0 +1,407 @@
+  primary username identities, this will be the same value as <InlineField>{props.base_field_name}.username</InlineField>.
+</APIField>
+
+<APIField name={props.base_field_name + '.identities[x].insertInstant'} type="Long" since="1.99.9">
+  The [instant](/docs/reference/data-types#instants) when the identity was created.
+</APIField>
+
+<APIField name={props.base_field_name + '.identities[x].lastLoginInstant'} type="Long" since="1.99.9">
+  The [instant](/docs/reference/data-types#instants) when the identity was last used to login. If a User has multiple identity types (`username`, `email`, and `phoneNumber`), then this value will represent the specific identity they last used to login. This contrasts with <InlineField>{props.base_field_name}.lastLoginInstant</InlineField>, which represents the last time any of the User's identities was used to login.
+
+  {/* more */}
+
+  <Aside type="note">
+    Identity provider logins will not be reflected here (see <InlineField>{props.base_field_name}.lastLoginInstant</InlineField> for a field that includes identity provider logins).
+  </Aside>
+</APIField>
+
+<APIField name={props.base_field_name + '.identities[x].lastUpdateInstant'} type="Long" since="1.99.9">
+  The [instant](/docs/reference/data-types#instants) when the identity was last updated.
+</APIField>
+
+<APIField name={props.base_field_name + '.identities[x].moderationStatus'} type="String" since="1.99.9">
+  <ModerationStatusResponse />
+
+  Only used for `username` type identities. For primary username identities, this will be the same value as <InlineField>{props.base_field_name}.usernameStatus</InlineField>.
+</APIField>
+
+<APIField name={props.base_field_name + '.identities[x].primary'} type="Boolean" since="1.99.9">
+  Reserved for future use.
+</APIField>
+
+<APIField name={props.base_field_name + '.identities[x].type'} type="String" since="1.99.9">
+  <IdentityTypeResponse />
+</APIField>
+
+<APIField name={props.base_field_name + '.identities[x].value'} type="String" since="1.99.9">
+  The value represented by the identity. The contents of this field depend on <InlineField>{props.base_field_name}.identities\[x].type</InlineField>and will be one of the following:
+
+  * Email address - The User's email address, always lower cased. For primary email identities, this will be the same value as <InlineField>{props.base_field_name}.email</InlineField>
+  * Phone number - The User's phone number, in E.164 canonical format. For primary phone number identities, this will be the same value as <InlineField>{props.base_field_name}.phoneNumber</InlineField>
+  * Username - User's username - this will be the same as <InlineField>{props.base_field_name}.identities\[x].displayValue</InlineField>. For primary username identities, this will be the same value as <InlineField>{props.base_field_name}.username</InlineField>
```

Comment:
I'm not sure it's the same as `displayValue` when unique usernames are enabled.

Reply:
Agreed it's not, see the `Aside` a couple lines below here. I was trying to figure out how to convey that simply, will making the happy path (most people don't use unique usernames) not too cluttered.